README.md

```diff
@@ -1,35 +1,27 @@
-# Louketo Proxy
+# Gatekeeper Proxy
```

Comment:
Shall we stick with "Gatekeeper"? So perhaps change all references of "Gatekeeper Proxy" to "Gatekeeper"

Reply:
it is still proxy...